### PR TITLE
Add pointers to SCDF guides in the Microsite

### PR DESCRIPTION
**Pull request by sabbyanandan**
The goal with this approach is to avoid duplicating content that we already maintain in the Microsite. And, of course, steer everyone towards the Microsite since that is the central source of truth for all the things SCDF and ecosystem.

### CODE REVIEW

**Comment by sabbyanandan** [comment]
Likewise, for SCSt: https://github.com/spring-guides/gs-spring-cloud-stream/pull/1, and SCT: https://github.com/spring-guides/gs-spring-cloud-task/pull/1

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+*.iws
+.idea/
+rebel.xml
+
+# Visual Studio Code
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```
Comment (dturanski) [inline .gitignore]: New line

File: README.adoc [hunk 2]
```diff
@@ -1 +1,44 @@
+== Getting Started with Spring Cloud Data Flow
+This guide walks you through an overview of Spring Cloud Data Flow and the process of orchestrating an event-driven
+streaming and short-lived batch data pipelines.
 
+=== What is Spring Cloud Data Flow?
+A microservices-based Streaming and Batch data processing in Cloud Foundry and Kubernetes. You can learn more about
+the Spring Cloud Data Flow from the link:https://spring.io/projects/spring-cloud-dataflow[project-site],
+link:https://spring.io/projects/spring-cloud-dataflow#learn[documentation],
+and link:https://github.com/spring-cloud/spring-cloud-dataflow-samples[samples].
```
Comment (dturanski) [inline README.adoc:9]: I think we should link to dataflow.spring.io home page somewhere here.

File: README.adoc [hunk 3]
```diff
@@ -1 +1,44 @@
...
+link:https://dataflow.spring.io/docs/batch-developer-guides/batch/data-flow-spring-batch/[Spring Cloud Data Flow with Batch Jobs]
+
+=== Spring Cloud Data Flow and Composed Task
+In the following guide, we demonstrate how to orchestrate a Composed Task with a combination of both a simple task
+and a batch job application.
+
+link:https://dataflow.spring.io/docs/batch-developer-guides/batch/data-flow-composed-task/[Spring Cloud Data Flow and Composed Task]
+
+== Summary
+Congratulations! You have learned Spring Cloud Data Flow's high-level overview, and you were able to build, deploy, and
```
Comment (dturanski) [inline README.adoc]: ! You have completed ...

**Comment by dturanski** [comment]
LGTM!

**Pull request merged**